playbooks/vars/controller.yaml

```diff
@@ -1,10 +1,11 @@
 ---
 # Controller parameters can be included into each playbook for test purposes
-# Your AppDynamics controller host, port account and access key
-controller_host_name: "lncontroller20113-2011-bj3r9l8f.appd-cx.com"
-controller_port: "8090"
+# Your AppDynamics controller host, port account and access key - These need to updated to your controller values:
+
+controller_host_name: "ansiblelabiogbole-2120-mgk5x9tp.appd-cloudmachine.com" #Update this to controller DNS
+controller_port: "8090" #Default port - change to other port if not default
 controller_account_name: "customer1" # Please add this to your Vault
-controller_account_access_key: "c5c7e458-bd5d-4046-85ba-099868f11869" # Please add this to your Vault
+controller_account_access_key: "467f1a9e-a73b-46b7-bede-c9caad8bb317" # Please add this to your Vault
```

Comment:
be careful about the keys, remove them in next check in
suggestion: create .local.controller.yml under /varsthat you are going to load values to playbooks from:

name: Include variables for the controller settings
include_vars: vars/.local.controller.yaml
and git-ignore these files:
.local.*

Reply:
Have removed the hard coded values and using a local settings now.